### DISCUSSION TITLE
Does it work offline? Without backend?

### DISCUSSION THREAD

**Discussion by abitrolly** [oramasearch/orama#186] [Q&A] [answered]
Does Lyra work when deployed on static web hosting?

**Comment by micheleriva** [comment, marked as answer]
Hi @abitrolly!
Yes, absolutely, Lyra is 100% capable of running in your browser.

You can just import it as:

```html
<html>
  <body>
    <script type="module">
      import { create, search, insert } from "https://unpkg.com/@lyrasearch/lyra@latest/dist/esm/src/lyra.js";

      // ...
    </script>
  </body>
</html>
```

and use it directly in any major browser.

Let me know if you have any other questions!

**Reply by abitrolly** [reply]
Yes. I wonder what are the limitations for browser side search?

Here https://abitrolly.github.io/fedora-packages-static/ I've got the listing of 65000 Fedora packages. Raw HTML for it is probably few gigabytes. So is it feasible to add this Lyra search to that static website? Package names are critical, then if possible short descriptions, and then full text.

**Reply by demosthenez** [reply]
I'm sorry but this isn't offline, right?  It depends on a cdn, no?
